### PR TITLE
[WIP] 9754 update jquery 2 part nbm

### PR DESCRIPTION
**Pull request by nmiddleweek**
For review.

I'll clear out the rogue commits tomorrow.

### CODE REVIEW

File: app/assets/javascripts/components/JQueryMigrateWrapper.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+     * 4.   jquerymigrate: ['jquerySrc'],
+     * 5.   jquery: ['jquerymigrate'],
+     *      .
+     *      .
+     *      .
+     *    }
+     *
+     */
+
+    console.log('jQuery.migrateVersion', jQuery.migrateVersion, 'jQuery.fn.jquery', jQuery.fn.jquery);
```
Comment (davidtrussler) [inline app/assets/javascripts/components/JQueryMigrateWrapper.js]: I can't see this being logged anywhere. Is it useful?

**Comment by davidtrussler** [comment]
@nmiddleweek I'm not sure why this is in a separate branch. I think it would be clearer to make the jQueryMigrateWrapper commit to the existing one. Also a fuller description would be useful to explain what this is doing.

File: app/assets/javascripts/components/JQueryMigrateWrapper.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+     *
+     *    shim: {
+     * 4.   jquerymigrate: ['jquerySrc'],
+     * 5.   jquery: ['jquerymigrate'],
+     *      .
+     *      .
+     *      .
+     *    }
+     *
+     */
```
Comment (davidtrussler) [inline app/assets/javascripts/components/JQueryMigrateWrapper.js]: Can we add a comment that this should be deleted when the migrate work is done, and also a comment in require_config to document that that should be updated also at the end of the process?

**Comment by willhall88** [comment]
@davidtrussler So I asked Nick to upload the new commit to a new branch, but I think it was lost in translation. I just intending on it being a single commit on it's own branch as I was going to then take that forward with the progression through 1.11 -> 1.12 -> 3 that we need to go through and cherry pick over some of the other commits as needed. Rather adding these changes (like rolling back to 1.12 etc.) to the existing branch

**Comment by willhall88** [comment]
We also found that adding the setup with the wrapper module should trigger the jqmigrate on all pages on frontend, which we weren't seeing on non tool pages. I will update in the channel too

**Comment by willhall88** [comment]
closing this in favour of #2076

**Pull request closed without merging**